brave/src/main/java/brave/internal/InternalPropagation.java

```diff
@@ -11,11 +11,26 @@
  * <p>Inspired by {@code okhttp3.internal.Internal}.
  */
 public abstract class InternalPropagation {
+  public static final int FLAG_SAMPLED = 1 << 1;
+  public static final int FLAG_SAMPLED_SET = 1 << 2;
+  public static final int FLAG_DEBUG = 1 << 3;
+  public static final int FLAG_SHARED = 1 << 4;
+  public static final int FLAG_SAMPLED_LOCAL = 1 << 5;
```

Comment:
Might be good to get a contextual comment about these flags in here

Reply:
done